Gordon360/Models/CCT/Housing/DueDate.cs

```diff
@@ -0,0 +1,19 @@
+﻿// <auto-generated> This file has been auto generated by EF Core Power Tools. </auto-generated>
+#nullable disable
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.EntityFrameworkCore;
+
+namespace Gordon360.Models.CCT;
+
+[Table("DueDate", Schema = "Housing")]
+public partial class DueDate
+{
+    [Key]
+    [Column("DueDate")]
+    [StringLength(255)]
+    [Unicode(false)]
+    public string DueDate1 { get; set; }
```

Comment:
Why is there a 1 at the end of this column name?

Reply:
I'm also confused about it! I don't have the 1 in my database, but the EF-Core add the 1 when it auto generates this file. Do you know the reason about it?